test/launchesReporting.test.js

```diff
@@ -0,0 +1,63 @@
+const EventEmitter = require('events');
+const { getDefaultConfig, RPClient, mockedDate } = require('./mocks');
+const RPReporter = require('./../lib/mochaReporter');
+describe('launch reporting', function() {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it('should start launch with default options', function() {
+    const options = getDefaultConfig();
+    const runner = new EventEmitter();
+    const reporter = new RPReporter(runner, options);
+    reporter.rpClient = new RPClient(options);
```

Comment:
What do you think about putting getDefaultConfig, new EventEmitter, new RPReporter, new RPClient into beforeEach method?

Reply:
@KatsiarynaTatarynovich 
I think this case cannot be applied here.
It seems we need different reporter and client instances with different options in each test case.